src/layout/Menu/examples/SideNav.tsx

```diff
@@ -0,0 +1,228 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+import { Button } from '../../../components/';
+import { Menu } from '../Menu';
+import { Panel } from '../../Panel/Panel';
+import { Home, Lock, Person } from '../../../icons';
+import { core } from '../../../tokens/';
+import { grayscale, white } from '../../../color';
+
+const StyledPanel = styled(Panel)`
+  ${core.edge(0)};
+  background-color: ${({ theme }) =>
+    theme.name === 'dark' ? grayscale(850) : white};
+`;
+
+const dividerCSS = css`
+  content: '';
+  height: 1px;
+  width: 100%;
+  left: 0;
+  position: absolute;
+  bottom: 0;
+  background-color: ${core.color.stroke};
+`;
+
+export function SideNav() {
+  const folders = (
+    <Menu
+      style={{
+        width: '100%',
+      }}
+      format="basic"
+    >
+      <MenuContainer>
+        <Menu.Item active={true} toggle icon={<Home />}>
+          {'Home'}
```

Comment:
Why are all of these strings formatted as JSX blocks of strings instead of normal strings?